runtime/src/lib.rs

```diff
@@ -65,6 +65,9 @@ use scale_info::TypeInfo;
 pub use sp_runtime::BuildStorage;
 pub use sp_runtime::{Perbill, Permill};
 
+/// litentry pallet
+pub use pallet_credit_score;
+
```

Comment:
Are we safe to delete this import?

Reply:
yes, but we can add it then expose everything defined in pallet_credit_score. you can see all pallets added from Integritee.
/// added by Integritee
pub use pallet_claims;
/// added by Integritee
pub use pallet_teeracle;
/// added by Integritee
pub use pallet_teerex;